scripts/check-g-cloud-live.py

```diff
@@ -2,7 +2,7 @@
 """
 
 Usage:
-    scripts/check-g-cloud-7-live.py <stage> <api_token> <draft_bucket> <documents_bucket>
```

Comment:
To be honest I'm not sure what it was for - looks like a thing Rob Young wrote to reassure himself he hadn't fucked things up copying a bunch of files over.

I'm gonna nuke it along with the others.